src/design-system/components/List/ListGroup.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+interface ListGroupProps {
+  children: React.ReactNode;
+  ordered?: boolean;
+}
+
+const ListGroup: React.FC<ListGroupProps> = ({ children, ordered = false }) => {
+  return (
+    <>
+      {ordered ? (
+        <ol className="list">{children}</ol>
+      ) : (
+        <ul className="list">{children}</ul>
+      )}
+    </>
+  );
+};
```

Comment:
Not entirely sure about the benefit of this component over simply exposing `ol` & `ul`. I fear this abstraction might bring about more confusion, for example:
- What's the difference between using the elements directly and the wrapper?
- What happens if some of my `children` are not `ListItem` components?
- Why is the default list unordered if I omit the attribute?

Reply:
Hey Max, super appreciate your review. I get where you are coming from, and I did have similar reservations when making that component and those are fair points about confusion around the component. 

For context, these were the factors I considered: 

 - In our base css we remove any listing styling a browser would add
 - We need unstyled lists in serveral places, e.g the filter, the nav, etc.
 - We will likely need lists in our about pages and new blog

My options were:

1. Add a css class for lists that we just add to lists where we need styled lists
2. Reintroduce default list styling by updating the base css and temporarily fix the places where that breaks styling
3. Create a custom component for a list

I dismissed the first because it would break the patterns we've established in this design system. Whilst we are using classes, this is very much a "Styled Component" inspired design system where we don't envisage developers having to add classes to loads of things. 

The second, I also dismissed because I assumed it would be messy to refactor, but I might have dismissed that to easily.

The third I choose because it was something used both on the `login` and `about` pages, and could see being used in the new blog styling. 

That was a super long winded way of saying, you've convinced me to remove this component and instead go for the second option and use `ol` & `ul`. 😆

Reply:
I second the second option as it'll be super useful for when we get blog posts going, right now the blog setup uses just standard markdown (not MDX) so having styles that work well for instances of `ol` and `ul` instead of having the hassle of inserting the components into the MDX will be great.

Reply:
I have messed with the timeline and like magic that List component never existed 🪄

I've returned us to default styling for lists, adding consistent margins using our spacing variables. Along the way, I addressed issues this caused in a few places—mainly parts I hadn’t yet refactored. So for now, I've updated the relevant styled components and plan to tidy this up properly when I come to them.